rama-tls/src/rustls/client/connector.rs

```diff
@@ -432,14 +432,11 @@ impl<S, K> TlsConnector<S, K> {
         let store_server_cert_chain = connector_data
             .is_some_and(|data| data.client_config_input.store_server_certificate_chain);
 
-        let server_certificate_chain = store_server_cert_chain
-            .then(|| {
-                conn_data_ref
-                    .peer_certificates()
-                    .map(|chain| chain.try_into().ok())
-            })
-            .flatten()
-            .flatten();
+        let server_certificate_chain = if store_server_cert_chain {
+            conn_data_ref.peer_certificates().map(DataEncoding::from)
```

Comment:
You can normally also do `map(Into::into)`, which is already in the prelude :)

Reply:
Cool trick thanks